bin/collator/src/evm_tracing_types.rs

```diff
@@ -120,4 +141,49 @@ pub struct EthApiOptions {
     /// Maximum number of logs in a query.
     #[clap(long, default_value = "10000")]
     pub max_past_logs: u32,
+
+    /// Sets the backend type (KeyValue or Sql)
+    #[clap(long, value_enum, ignore_case = true, default_value_t = FrontierBackendType::default())]
+    pub frontier_backend_type: FrontierBackendType,
+
+    /// Sets the SQL backend's pool size.
+    #[arg(long, default_value = "100")]
+    pub frontier_sql_backend_pool_size: u32,
+
+    /// Sets the SQL backend's query timeout in number of VM ops.
+    #[clap(long, default_value = "10000000")]
+    pub frontier_sql_backend_num_ops_timeout: u32,
```

Comment:
Question: What happens if you set **0** here? I am asking to make sure we are not missing validation of input params.

Reply:
When creating a new SQL instance in `open_frontier_backend` it acually ensure it's non zero using `std::num::NonZeroU32::new(num_ops_timeout)`

Check [here](https://github.com/AstarNetwork/Astar/blob/fd8e81c6495e9237a412de4c3568cc5b55d56e96/bin/collator/src/rpc.rs#L127)

Reply:
Perfect then 👍